.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### Before contributing
+
+Please review the [contributing section](https://urllib3.readthedocs.io/en/latest/contributing.html) in the documentation.
+
+### Checklist
+
+- [ ] (For bugfixes) Write tests
```

Comment:
Honestly I don't know if we need much in our PR template aside from pointing to our contributing documentation and saying to review it before submitting a PR. That way we can make all the updates to our contributing documentation :)

Reply:
Removed :)